starter-code/src/viking.js

```diff
@@ -1,11 +1,103 @@
 // Soldier
-function Soldier() {}
-
-// Viking
-function Viking() {}
-
-// Saxon
-function Saxon() {}
-
-// War
-function War() {}
+function Soldier (health, strength) {
+    this.health = health;
+    this.strength = strength;
+  };
+  
+  
+  Soldier.prototype.attack = function () {
+    return this.strength;
+  };
+  
+  Soldier.prototype.receiveDamage = function (damage) {
+    this.health =  this.health - damage;
+  };
+  
+  // Viking
+  function Viking(name, health, strength) {
+    Soldier.call(this, health, strength);
+    this.name = name;
+  };
+  
+  Viking.prototype = Object.create(Soldier.prototype);
+  Viking.prototype.constructor = Viking;
+  
+  Viking.prototype.receiveDamage = function(damage) {
+    this.health = this.health - damage;
+    if (this.health > 0) {
+      return this.name + ' has received ' + damage + ' points of damage';
+    } else {
+      return this.name + ' has died in act of combat';
+    }
+  };
```

Comment:
De esta manera machacais la función que habéis heredado en vez de reutilizarla. Debéis modificar la función heredada de la siguiente manera -> haced `Soldier.prototype.receiveDamage.call(this, damage)` dentro de vuestro recién creado `Viking.prototype.receiveDamage(damage)`